test/e2e/k8s_util.go

```diff
@@ -333,6 +336,44 @@ func decidePingProbeResult(stdout string, probeNum int) PodConnectivityMark {
 	return Error
 }
 
+func (k *KubernetesUtils) digUsingShort(
+	podName string,
+	podNamespace string,
+	dstAddr string,
+	useTCP bool,
+	dnsServiceIP string) (string, error) {
+
+	// Get the Pod
+	pod, err := k.clientset.CoreV1().Pods(podNamespace).Get(context.TODO(), podName, metav1.GetOptions{})
```

Comment:
It is not worth it to get the Pod this way just to get the container name. You should have the container name as a function parameter.